src/components/content/credential/CredentialTip.tsx

```diff
@@ -0,0 +1,35 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ * SPDX-FileCopyrightText: Huawei Inc.
+ */
+
+import { Alert } from 'antd';
+import React from 'react';
+
+export const CredentialTip = ({
+    type,
+    msg,
+    onRemove,
+}: {
+    type: 'error' | 'success' | undefined;
+    msg: string;
+    onRemove: () => void;
+}): JSX.Element => {
+    if (!type) {
+        return <></>;
+    }
+
+    return (
+        <div className={'monitor-tip-class'}>
+            {' '}
+            <Alert
+                className={''}
+                message='Credential:'
```

Comment:
For all the actions - 
- main message must be "Credential added/deleted/updated" based on the action.
- Description can be removed.

Reply:
result  alert message must be on the modal itself. Otherwise it is not clear to the user.

Reply:
still the result message is displayed outside modal. We must free the form and then display the response Alert inside the modal itself.